### DISCUSSION TITLE
Block Config in WebGUI

### DISCUSSION THREAD

**Discussion by nsauter** [blakeblackshear/frigate#10953] [General]
Hello!

I just wanted to ask one thing here: I dont need authentication on frigate but it would be great that users are not able to edit configuration params on the Web since they can break the whole setup without knowing it. Is there an easy way to make the config read-only or completely invisible?

Thanks!

**Comment by pcislocked** [comment]
hi, did you find any solution? I'm thinking of disabling POST requests on the proxy I'm using for external access, but don't feel like it's the best solution

**Reply by nsauter** [reply]
Nope. I Did not. But i havent tried to be honest. Maybe (if you Are using it behind a web proxy) you Could disable the slugs for the config page. Maybe this will result in a web error but better than destroying the whole config.

**Reply by pcislocked** [reply]
> Nope. I Did not. But i havent tried to be honest. Maybe (if you Are using it behind a web proxy) you Could disable the slugs for the config page. Maybe this will result in a web error but better than destroying the whole config.

i might try blocking all post requests altogether + the disabling slugs thing you said, it's behind cloudflared

**Reply by chrislawso** [reply]
I found your posts after I already submitted a similar new post here https://github.com/blakeblackshear/frigate/discussions/15545

So far I have tried configuring caddy with the following and this only restricts new page loads to /config and does not stop access from the frigate webgui.
```
@blocked {
    path /config
}
respond @blocked 403
```

What proxy are you using and have you found any successful methods?

Thanks

**Reply by pcislocked** [reply]
i didnt honestly, just told my dad and mom to not click it and then i forgot about it

**Reply by chrislawso** [reply]
@Firerouge found solution at https://github.com/blakeblackshear/frigate/discussions/15545#discussioncomment-11998647
